### ISSUE TITLE
cron job for cleaning up background processes

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/opentree#845]
See https://github.com/OpenTreeOfLife/opentree/pull/763

**Comment by jar398** [comment]
Another way to do this would be from an rc.d script, which we could write.  This would have the advantage that we could at the same time also take care of starting the neo4j services.  So, before dealing with cron, consider (and rule out) writing the rc script.  cron is admittedly easier.

**Comment by jimallman** [comment]
> Another way to do this would be from an rc.d script, which we could write.

Agreed, we only need to launch this script (`sessions2trash.py`) once, and it will periodically do the cleanup. But this script needs to be run in the proper context (incl. effective user, python executable, virtualenv, web2py application, etc.), so we'll need to start and kill a separate instance of it (or perhaps a well-named shell script that sets this context) for each web2py application on a deployed system. Working out the details for this now...

Hm, I see that ot20 (api.opentreeoflife.org) has its `/etc/inittab` set to start in runlevel 2, but it seems we're currently at runlevel 5. Is this a deliberate switch after reboot? If so, what moves us from runlevel 2 to 5? The services listed in both `/etc/rc#.d` directories look the same at a glance. Hm, it looks like [Debian treats runlevels 2-5 as identical](https://wiki.debian.org/RunLevel), so I can just start this in runlevels 2-5 and stop it in 0, 1, and 6. (This is the current setup for apache2.)

Also, note that api.opentreeoflife.org is running Debian 3.2.65 (wheezy) and using systemd instead of init, but it still respects traditional init scripts, so I'll keep working on this.
